src/zenml/integrations/aws/flavors/sagemaker_step_operator_flavor.py

```diff
@@ -29,17 +31,68 @@
 class SagemakerStepOperatorSettings(BaseSettings):
     """Settings for the Sagemaker step operator.
 
-    Attributes:
-        instance_type: The type of the compute instance where jobs will run.
-            Check https://docs.aws.amazon.com/sagemaker/latest/dg/notebooks-available-instance-types.html
-            for a list of available instance types.
+    Non-Estimator attributes:
         experiment_name: The name for the experiment to which the job
             will be associated. If not provided, the job runs would be
             independent.
-    """
+        input_data_s3_uri: S3 URI where training data is located if not locally,
+            e.g. s3://my-bucket/my-data/train. How data will be made available
+            to the container is configured with input_mode. Two possible
+            input types:
+                - str: S3 location where training data is saved.
+                - dict[str, str]: (ChannelName, S3Location) which represent
+                    channels (e.g. training, validation, testing) where
+                    specific parts of the data are saved in S3.
+
+    Estimator attributes:
+        See https://sagemaker.readthedocs.io/en/stable/api/training/estimators.html#sagemaker.estimator.Estimator
+        for a full list of attributes.
 
+    """
+    instance_count: Optional[int] = None
```

Comment:
Maybe a small suggestion here: These attributes will need constant updates and might not even be consistent across the Sagemaker versions that are supported by ZenML. Do you think it would be an option to just make this one attribute `estimator_args: Dict[str, Any]` which ZenML directly forwards the the Estimator initialization?